### PR TITLE
update: set the branch of each component to match odh nightly

### PR DESCRIPTION
**Pull request by zdtsw**
## Description

to match what we have for odh-nightly so we build locally the same as the odh-nightly produced

i have another internal PR 1756 but this one is mainly reflected thats being used there 
for dashboard: https://redhat-internal.slack.com/archives/C05SMJ09DD2/p1709805946505909

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

**Comment by zdtsw** [comment]
on hold this, also added component leads to get confirmed

**Comment by ruivieira** [comment]
LGTM, thanks!

**Review by ruivieira — APPROVED** [review]

**Comment by zdtsw** [comment]
dashboard will transit to main branch in odh 2.10 (next week) lets wait for a couple of days before merge it

File: get_all_manifests.sh
Comment (israel-hdez) [inline get_all_manifests.sh:18]: If I'm not wrong, for model-mesh this is still expected to be `release-0.11.1`.
If the nightly is using `main`, I think we should fix the nightly.

Comment (zdtsw) [reply get_all_manifests.sh:18]: leave this to you and your team to decide if keep "release-0.11.1"
- either we fix in midstream PR, (switch to release-0.11.1)
- or apply this PR change here

but i think you and team need to consider to have a dedicate branch for nightly and tags for odh releases in general (as other teams are doing)

Comment (israel-hdez) [reply get_all_manifests.sh:18]: We do incubate code, but the incubation branch is typically promoted from incubation quickly.
I'll confirm on Monday.

Comment (zdtsw) [reply get_all_manifests.sh:18]: https://github.com/opendatahub-io/modelmesh-serving/branches
these are the availbe branches: main, release-0.11.1, 
i cannot find "incubating" @Jooho which you want to use, this is not for release, this is for odh-nightly build

Comment (israel-hdez) [reply get_all_manifests.sh:18]: Our incubating release is, for now, the `release-0.11.1` branch :-)
So, if I understood Jooho, we do want to keep here model-mesh unchanged and fix odh-nighly.

Comment (zdtsw) [reply get_all_manifests.sh]: changes reverted

File: get_all_manifests.sh
Comment (andrewballantyne) [inline get_all_manifests.sh:13]: Incubation is still our ODH Nightly branch -- this is a premature change. We are not ready to chase down all the threads and make this change. Please revert this back to `incubation`.

Comment (zdtsw) [reply get_all_manifests.sh:13]: ok, i will not have the change for dashboard, has been waiting for weeks and i will get trusty out now.

**Pull request merged**
